### ISSUE TITLE
Using archive no longer works

### ISSUE THREAD

**Issue by DasIgno** [caarlos0/twitter-cleaner#51]
First of all, thank you for that great tool.

Unfortunately twitter seems to have changed the structure of the archives. I don't know, when that happened, until now I never had the need for tools like this. Anyway, in my archive, which is a few days old, there is no data/tweet.js, so twitter-cleaner fails. There are tweet-headers.js and tweets.js in the data folder, but the inner structure doesn't match the former tweet.js. I tried renaming both files to tweet.js, but ran into the error:
```
failed to delete tweets from archive error=invalid character 'w' looking for beginning of value
```

I would be really happy, if the archive-part of your tool would work again. Especially these stormy days. Tell me, if you need more info. I'll do my best to provide.

**Comment by DasIgno** [comment]
Manually applied #52, works fine. Thanks @bl1nk.

**Comment by greenoid** [comment]
Yeah, me too. Manually changed tweet to tweets two times and compiled again. Works. Thanks @bl1nk!
